README.md

```diff
@@ -201,6 +201,42 @@ El resultado entregado contiene la confirmación de la anulación, en la forma d
 Esta librería usa [Project Lombok][lombok] en su desarrollo. Si bien no es necesario podrías querer instalar el [plugin][lombok-plugins]
 para tu IDE favorito con el fin de evitar que veas errores marcados por la herramienta de desarrollo.
 
+## Deploy manual a maven central
+
+Agregar en el setting.xml 
```

Comment:
Where is this settings file? Is it .m2/settings.xml? In that case, be explicit about the path.

Reply:
@afiebig I don't think you have make this line clearer on the latest changes